pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1246,6 +1246,9 @@ func validateVolumes(field *k8sfield.Path, volumes []v1.Volume, config *virtconf
 		if volume.Secret != nil {
 			volumeSourceSetCount++
 		}
+		if volume.DownwardAPI != nil {
+			volumeSourceSetCount++
```

Comment:
I think this deserves a unit test.

Reply:
@rmohr I'm not sure I understand.  What exactly to test ?   Why is it needed only for the downwardAPI case and not for all other volume types ?

Reply:
I thought that this does something different. All good. It would make sense to increase the coverage for these things here in a more generic way. Outside of the scope of this PR.